src/components/AddComment.jsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react"
+import { postCommentByArticleID } from "../api";
+
+const AddComment = ({article_id, setIsListed}) => {
+
+    const [clicked, setIsClicked] = useState(false)
+    const [form, setForm] = useState({});
+    const [isListing, setIsListing] = useState(false);
+    const [disabled, setDisabled] = useState(true)
+
+    const handleClick = () => {
+      setIsClicked((prevState)=> !prevState)
+    }
+
+    const handleChange = (event) => {
+      setForm((currentForm) => {
+        const information = { ...currentForm, [event.target.id]: event.target.value }
+        const username = information.username
+        const body = information.body
+        if (username === undefined || body === undefined || body.length === 0) {
```

Comment:
This is nice logic although you could just set the body and username inputs to 'required' for cleaner code!